src/objects/Opportunity.object

```diff
@@ -855,6 +855,19 @@
         <trackTrending>false</trackTrending>
         <type>Checkbox</type>
     </fields>
+    <fields>
+        <fullName>Elevate_Recurring_ID_c__c</fullName>
+        <description>The external, unique ID of the Recurring Donation in Elevate that this donation Opportunity is linked to.</description>
+        <externalId>true</externalId>
+        <inlineHelpText>The external, unique ID of the Recurring Donation in Elevate that this donation Opportunity is linked to.</inlineHelpText>
+        <label>Elevate_Recurring_ID__c</label>
```

Comment:
@callen-sfdo , @pedrodalcol : Is this new field the "Elevate Recurring Id` (`CommitmentId__c`) ? For the RD2 purposes, we added [the field](https://github.com/SalesforceFoundation/NPSP/blob/5ecc4123389f4e82622e2813f7e2bc0d428ed005/src/objects/Opportunity.object#L853), but it is in the `feature/RD` branch (not to be merged in this sprint). See PR #5359.

Reply:
@force2b  ^

Reply:
@RadSFDeveloper we need to merge the field in this release or we will be blocked

Reply:
We're going to hold this PR for discussion Monday morning to ensure that we get all the appropriate consideration and eyes on it.